packages/plugins/graphql/server/services/builders/resolvers/component.js

```diff
@@ -13,6 +13,9 @@ module.exports = ({ strapi }) => ({
       const component = strapi.getModel(componentName);
 
       const transformedArgs = transformArgs(args, { contentType: component, usePagination: true });
+      await validate.contentAPI.query(transformedArgs, contentType, {
```

Comment:
Maybe I got lost in the PR but, what happens if no auth is provided?

Reply:
The same as sanitize. It throws on all the invalid params, but it won't check permission on them.